### PR TITLE
Update About page

### PR DESCRIPTION
**Pull request by khoeger**
- modify time ranges
- emphasize that anyone can go
- fix links
- remove old hacker league group information
- fix quote

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Programmers, engineers, and scientists with an interest in any aspect of music.
 
 ## What can I make?
 
-Wiktionary gives one definition of hack as “To work on an intimately technical level”  (http://en.wiktionary.org/wiki/hack).  An appropriate hack at this event would work on an intimately technical level with any subject surrounding music, such as making sound, making instruments, making compositions, analysis of a piece of music, analysis of a music collection, analysis of music listening data, making music presentation platforms, making music sharing apps, making music sales apps, etc, etc, etc…
+Wiktionary gives one definition of [hack](http://en.wiktionary.org/wiki/hack) as “To work with on an intimately technical level.”  An appropriate hack at this event would work on an intimately technical level with any subject surrounding music, such as making sound, making instruments, making compositions, analysis of a piece of music, analysis of a music collection, analysis of music listening data, making music presentation platforms, making music sharing apps, making music sales apps, etc, etc, etc…
```
Comment (trevorknight) [inline about.md]: "To work with on"?

**Review by trevorknight — COMMENTED** [review]
Hmmm Looking at the diffs, it shows changes that are already in master, for example, the list of people or sponsors. (Take a look at mmhpreview.com/about---they're not there.)

Looks like you committed on top of my unmerged commit in the 'about' branch. Did you do that on purpose?

File: about.md (hunk 1 above)
Comment (trevorknight) [reply about.md]: Um, I updated the definition: "To work with something on an intimately technical level"

Comment (khoeger) [reply about.md]: 1. I thought I had updated the about to the master branch before making revisions. Was I wrong? I thought the revisions were being made on top of what was on the current mmhpreview.com master about.md
2. I made a commit mistake. I checked out the about branch, and somehow switched to my local master and committed my changes to that, then made a pull request on the about branch, which hadn't undergone changes. 
I fixed it by
- reverting master to the previous commit, then pushing
- copying my prior changes to the master's about.md to the about branch's about.md
- commiting and pushing the new about.
3. There are more changes. I don't know if you agree with all of them. That was one of them. The rest have to do with the time frame, how we display links, etc.

**Comment by trevorknight** [comment]
You can see on this page, the two commits:
https://github.com/musichackathon/mmh_jekyll/pull/75/commits

Probably what happened was:
- You had the old about branch from july on your computer
- You checked out that branch rather than creating a new branch
- Committed on top of it.

You could fix this by using interactive rebase:
$ git rebase -i
Then change the 'pick' next to my commit to 'drop'
Save and exit

Yeah, the stuff about the hack time being shorter and it ending at 9 not 10
is fantastic. Also, the links are great.

**Review by trevorknight — APPROVED** [review]

File: about.md (hunk 1 above)
Comment (trevorknight) [reply about.md]: Cool. Just update the definition: "To work with something on an ..."

**Pull request merged**